package.json

```diff
@@ -17,6 +17,7 @@
     "ioredis": "^5.0.6",
     "lru-cache": "^7.10.1",
     "mdx-bundler": "^9.0.1",
+    "moment": "^2.29.3",
```

Comment:
can we please use `date-fns` apis instead of moment? date-fns is already in the project, and `moment` is a dead project now

Reply:
Yes. `date-fns` is the date library we'll use across the project for all date-formatting and manipulation. Pete is correct. Moment has had it's moment, and that moment has passed ;)